misk-aws-dynamodb-testing/src/main/kotlin/misk/aws/dynamodb/testing/DockerDynamoDb.kt

```diff
@@ -52,6 +52,7 @@ object DockerDynamoDb : ExternalDependency {
     withImage("amazon/dynamodb-local")
         .withName(id)
         .withExposedPorts(exposedClientPort)
+        .withCmd("-jar", "DynamoDBLocal.jar", "-sharedDb")
```

Comment:
could you add a comment, perhaps linking to documentation, on why KCL depends on this option?

Reply:
I can add a comment, but AFAIK there is no AWS documentation around this. We figure this out when I was discussing this with platform team and from some try and error.